test/plugin.test.js

```diff
@@ -25,9 +25,7 @@ const externalSchemas2 = Object.freeze({
   }
 })
 
-const fastifyFjsOptionsDefault = Object.freeze({
-  customOptions: {}
-})
+const fastifyFjsOptionsDefault = Object.freeze({})
```

Comment:
strange. freeze is not recursive afaik. So customOptions could have always been modified?

Reply:
This was a wrong copy-paste from `ajv-compiler`. `customOptions` is an ajv setting